modules/rotate.html

```diff
@@ -0,0 +1,355 @@
+<!-- <?php
+include('../wp-blog-header.php');
+
+global $user_identity;
+if($user_identity){
+	echo $user_identity;
+}
+else{
+	//die('Sorry, you must be <a href="'. get_bloginfo('home') . '/wp-login.php?redirect_to=' . $_SERVER['PHP_SELF'] . '">logged in</a> to view this page.');
+}
+//echo do_shortcode( '[contact-form-7 id="1234" title="Contact form 1"]' );
+?> -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Rotating</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+    <link rel="shortcut icon" type="image/png" href="img/logo_webTab.png" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
```

Comment:
Another thing to look at for future work is updating the versions on these imports.  Boostrap is on version 5 and version 3 came out over 10 years ago!